tools/golden_tests_harvester/bin/golden_tests_harvester.dart

```diff
@@ -41,23 +41,18 @@ Future<void> main(List<String> args) async {
   }
 
   final io.Directory workDirectory = io.Directory(rest.single);
-  final AddImageToSkiaGold addImg;
-  final bool dryRun = results['dry-run'] as bool;
-  if (dryRun) {
+  final bool isDryRun = results['dry-run'] as bool;
+  late Harvester harvester;
```

Comment:
Just make this just plain `final Harvester harvester` and rely on Dart understanding the basic if-statement (I believe), if not then `late final`.

Reply:
done